CesiumGltf/include/CesiumGltf/AccessorView.h

```diff
@@ -191,6 +191,21 @@ template <class T> class AccessorView final {
    */
   int64_t size() const noexcept { return this->_size; }
 
+  /**
+   * @brief Returns the stride (number of bytes per component) of this accessor.
```

Comment:
I don't think "number of bytes per component" is quite accurate. It's actually the number of bytes from one value to the next, which can be greater than the number of bytes of each element if other elements are interleaved. So how about:

```suggestion
   * @brief Returns the stride of this accessor, which is the number of bytes from the start of one element to the start of the next.
```